centreon/www/class/centreonLDAP.class.php

```diff
@@ -1078,7 +1078,7 @@ private function setErrorHandler(): void
      * @param string $dn
      * @return string|bool
      */
-    private function getCnFromDn($dn)
+    private function getCnFromDn($dn): string|bool
```

Comment:
In PHP 8.2 you can type with true or false

```suggestion
    * @return string|false
    */
    private function getCnFromDn($dn): string|false
```

Reply:
since it needs to be back ported to 24.04 that is running on 8.1 I was  going to keep the bool type but you got my curiosity and the false type is already available in 8.0 and true has been added in 8.2. So since every function is only returning false and never true it will be compatible with the 24.04 version.

Thanks a lot for the tip!